src/components/molecules/ImageCarousel/ImageCarousel.tsx

```diff
@@ -20,90 +23,88 @@ const ImageCarousel = ({ images }) => {
     setSelectedImageIndex(newIndex);
   };
 
+  const [isImageVisible, setImageVisibility] = useState(true);
+
   return (
-    <>
-    <h1 className={styles.gameTitle}>City Tank</h1>
-    <div className={styles.gameShow}>
+    <div className={home.pageContainer}>
+      <h1 className={styles.gameTitle}>City Tank</h1>
+      <div className={styles.gameShow}>
 
-      <div className={styles.showCarousel}>
-        <div className={styles.mainShowContainer}>
+        <div className={styles.showCarousel}>
+          <div className={styles.mainShowContainer}>
 
-          {selectedImageIndex === 0 ? (
-            <iframe width="100%" height="100%" src="https://www.youtube.com/embed/XPKtuM19-VQ?si=emdi1l0Gu35Tw0aQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
-          ) : (
-            <img
-              src={images[selectedImageIndex]}
-              alt={`Image ${selectedImageIndex}`}
-            />
-          )}
-          {selectedImageIndex === 0 && (
-            <img
-              src={images[selectedImageIndex]}
-              alt={`Image ${selectedImageIndex}`}
-            />
-          )}        
-        </div>
-        <div className={styles.thumbnailCarousel}>
-          <button className={styles.prev} onClick={handlePreviousImage}>&lt;</button>
-          {images.map((image, index) => (
-            <div key={index}>
+            {selectedImageIndex === 0 ? (
+              <video controls width="100%" height="100%">
+              <source src="https://firebasestorage.googleapis.com/v0/b/pixels-forge-studio.appspot.com/o/games%2Fcity-tank%2FCity%20Tank%20-%20Compress.mp4?alt=media&token=1381b297-7fe0-4267-ab1c-fcec688ea0b5" type="video/mp4" />
+              Seu navegador não suporta o elemento de vídeo.
```

Comment:
What that means?

Reply:
This part is so that in the games carousel the first image can be a video if there is one, this image will be superimposed on the video and when it is clicked the video will appear in autoplay